tests/foreman/ui/test_errata.py

```diff
@@ -1297,54 +1284,50 @@ def test_positive_show_count_on_host_pages(session, module_org, registered_conte
 
 
 @pytest.mark.tier3
+@pytest.mark.rhel_ver_match('8')
 @pytest.mark.parametrize(
-    'module_repos_collection_with_setup',
-    [
-        {
-            'distro': 'rhel7',
-            'SatelliteToolsRepository': {},
-            'RHELAnsibleEngineRepository': {'cdn': True},
-            'YumRepository': {'url': settings.repos.yum_9.url},
-        }
-    ],
+    'registered_contenthost',
+    [[CUSTOM_REPO_URL]],
     indirect=True,
 )
-def test_positive_show_count_on_content_host_details_page(
-    session, module_org_with_parameter, erratatype_vm
+def test_positive_check_errata_counts_by_type_on_host_details_page(
+    session,
+    module_org,
+    registered_contenthost,
 ):
-    """Errata count on Content host Details page
+    """Errata count on host page
 
-    :id: 388229da-2b0b-41aa-a457-9b5ecbf3df4b
+    :id: 89676641-2614-4abb-afed-5c37be396fad
 
     :Setup:
-
         1. Errata synced on satellite server.
         2. Some content hosts are present.
 
-    :steps: Go to Hosts -> Content Hosts -> Select Content Host -> Details page.
-
-    :expectedresults: The errata section should be displayed with Security, Bug fix, Enhancement.
+    :steps:
+        1. Go to All hosts
+        2. Select the host
+        3. Content Tab -> Errata Tab
+        4. Check the counts of the errata types
 
-    :BZ: 1484044
+    :expectedresults: There should be correct number of each errata type shown.
     """
-    vm = erratatype_vm
+
+    vm = registered_contenthost
     hostname = vm.hostname
     with session:
         session.location.select(loc_name=DEFAULT_LOC)
-        content_host_values = session.contenthost.read(hostname, 'details')
-        for errata_type in ('security', 'bug_fix', 'enhancement'):
-            assert int(content_host_values['details'][errata_type]) == 0
+        read_errata = session.host_new.get_details(hostname, 'Content.Errata')
+        assert int(len(read_errata['Content']['Errata']['pagination'])) == 0
 
         pkgs = ' '.join(FAKE_9_YUM_OUTDATED_PACKAGES)
-        assert vm.execute(f'yum install -y {pkgs}').status == 0
-
-        # navigate to content host main page by making a search, to refresh the details page
-        session.contenthost.search(hostname)
-        content_host_values = session.contenthost.read(hostname, 'details')
-        assert int(content_host_values['details']['security']) == FAKE_9_YUM_SECURITY_ERRATUM_COUNT
-
-        for errata_type in ('bug_fix', 'enhancement'):
-            assert int(content_host_values['details'][errata_type]) == 1
+        assert vm.execute(f'yum install -y {pkgs}').status == 0 == 0
```

Comment:
Single compare should be enough.
```suggestion
        assert vm.execute(f'yum install -y {pkgs}').status == 0
```